modules/plugin-search.php

```diff
@@ -463,7 +471,7 @@ class="jetpack-plugin-search__primary button"
 			$links[] = '<a
 				id="plugin-select-settings"
 				class="jetpack-plugin-search__primary button jetpack-plugin-search__configure"
-				href="' . esc_url( $plugin['configure_url'] ) . '"
+				href="' . $this->get_configure_url( $plugin['module'], $plugin['configure_url'] ) . '"
```

Comment:
Not a huge fan of escaping within this function instead of right here on display as it could introduce an issue if `get_configure_url` is refactored in the future and we don't patch up the escaping right, but since it is escaped I don't want to block on that.